pkg/platform/runtime/setup/setup.go

```diff
@@ -160,6 +165,13 @@ func NewWithModel(target Targeter, msgHandler Events, model ModelProvider, an an
 
 // Update installs the runtime locally (or updates it if it's already partially installed)
 func (s *Setup) Update() error {
+	if s.target.InstallFromDir() != nil {
```

Comment:
Please add comment referencing what the distinction is, what the temporary nature of this is, and a link to https://activestatef.atlassian.net/browse/DX-846